client/js/util.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Delays an async function.
+ *
+ * @param {number} ms The number of milliseconds to wait before resolving the
```

Comment:
this file is git pulled from master

Reply:
If it's already in master why add it to this pull request?

Reply:
I needed it in my local files so I can see the results.

It's not adding extra files, I just pulled it from master.

Reply:
yeah, it's just showing up in this PR b/c you didn't merge master into the branch this PR is based on. you should probably just do that, though.

Reply:
^

Reply:
I will git pull everything from master